clox/functions.py

```diff
@@ -231,6 +231,12 @@ def run_clock(
         if not hide_timezone:
             print(" " * h_shift, end='')
             print("Timezone: {timezone}".format(timezone=timezone_str))
+            if tz is not None:
```

Comment:
Why is it required? And it seems it's the same as `if timezone is not None`, if so, would you change it to that? It's more interpretable.

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/54/commits/e28d1238647ea6a8883e9e619096a085c31b964d